packages/widgets/src/components/ExchangeAccountKeys/ExchangeAccountKeys.spec.tsx

```diff
@@ -0,0 +1,42 @@
+import userEvent from '@testing-library/user-event';
+import { StoryComponent } from './stories/StoryComponent';
+
+describe('ExchangeAccountKeys', () => {
+  it('Renders with title', () => {
+    const { getByText } = render(<StoryComponent onDelete={jest.fn()} />);
+
+    getByText('Account Name');
+  });
+
+  it('Renders with delete button', async () => {
+    const user = userEvent.setup();
+    const mockOnClick = jest.fn();
+    const { getByRole } = render(<StoryComponent onDelete={mockOnClick} />);
+
+    const deleteButton = getByRole('button', { name: 'Bin' });
+    await user.click(deleteButton);
+    expect(mockOnClick).toBeCalledTimes(1);
+  });
+
+  it('Renders with API key', () => {
+    const { getByText } = render(<StoryComponent onDelete={jest.fn()} />);
+
+    getByText('123456789012345678901234567890');
```

Comment:
Check there's `***` text present in the component

Reply:
Done